packages/components/src/InputSelect/Component.purs

```diff
@@ -0,0 +1,99 @@
+module KSF.InputSelect.Component where
+
+import Prelude
+
+import Data.Array (snoc)
+import Data.Foldable (foldMap)
+import Data.Generic.Rep (class Generic)
+import Data.Generic.Rep.Show (genericShow)
+import Data.Maybe (Maybe(..), fromMaybe, isJust, isNothing)
+import Data.String (toLower)
+import Effect (Effect)
+import React.Basic (JSX)
+import React.Basic as React
+import React.Basic.DOM as DOM
+import React.Basic.DOM.Events (targetChecked)
+import React.Basic.Events (handler)
+
+type Self = React.Self Props State
+
+type Props =
+  { type_           :: InputType
+  , name            :: String
+  , value           :: Maybe String
+  , checked         :: Maybe Boolean
+  , onChange        :: Maybe Boolean -> Effect Unit
+  , label           :: Maybe String
+  , required        :: Maybe Boolean
+  }
+
+type State =
+  { checked :: Boolean }
+
+data InputType = Radio | Checkbox
+
+derive instance genericInputType :: Generic InputType _
+instance showInputType :: Show InputType where
+  show = toLower <<< genericShow
+
+component :: React.Component Props
+component = React.createComponent "InputSelect"
+
+inputSelect :: Props -> JSX
+inputSelect = React.make component
+  { render, didMount, initialState }
+  where
+    didMount { props, setState } = when (isJust props.checked) $
+      setState \s -> s { checked = fromMaybe false props.checked }
```

Comment:
I think this bit could be improved. You are calling `setState` when `props.checked` contains something, but then you use `fromMaybe` on it. I think this is an undesired consequence of having `checked` wrapped in a `Maybe`. Can't it be a simple `Boolean` who is assigned `False` when instantiating the component?